src/commonMain/kotlin/fr/acinq/lightning/serialization/v4/Deserialization.kt

```diff
@@ -64,30 +66,54 @@ object Deserialization {
         lastSent = readLightningMessage() as ChannelReady
     )
 
+    private fun Input.readWaitForFundingSignedWithPushAmount(): WaitForFundingSigned {
+        val channelParams = readChannelParams()
+        val signingSession = readInteractiveTxSigningSession()
+        // We previously included a local_push_amount and a remote_push_amount.
+        readNumber()
+        readNumber()
```

Comment:
At this state, the initial `LocalCommit`/`RemoteCommit` objects were already fully generated, and we don't need local/remote `push_msat` to validate signatures.

We would only need local/remote `push_msat` if we were to do a rbf, which we are not doing in Phoenix due to zeroconf.

So we can safely discard those values.

Reply:
Those are just review notes confirming that it is okay to discard those two fields after reading, which is not obvious. In particular it wasn't obvious that we wouldn't need this information to validate the signatures received in that state.